app/models/services/index.ts

```diff
@@ -83,6 +93,18 @@ function validateServiceConfig(config: ServiceConfig<unknown>): void {
   }
 }
 
+/**
+ * Throws an error if the configuration is invalid. Logs a warning if configuration will be ignored.
+ * @param config - The service configuration to validate
+ */
+export function validateServiceConfig(config: ServiceConfig<unknown>): void {
+  if (config.umm_s === undefined || typeof config.umm_s !== 'string') {
+    throw new RequestValidationError(`There must be one and only one umm_s record configured as a string for harmony service: ${config.name}`);
```

Comment:
RequestValidationError is meant to be thrown from an HTTP call and return a 400 status code to a user along with a message.

I'm not sure if there's an existing exception more suited for this - maybe an InternalServerError is defined? If not you can just add a new Exception class in `errors.ts`.

Reply:
I guess we could use the existing `ServerError`, but in this case, the server is not up, so the 500 status code will not be returned to the client.